src-self-hosted/test.zig

```diff
@@ -64,10 +64,11 @@ pub const TestContext = struct {
         /// such as QEMU is required for tests to complete.
         target: std.zig.CrossTarget,
         /// In order to be able to run e.g. Execution updates, this must be set
-        /// to Executable.
+        /// to Executable. This is ignored when generating C output.
```

Comment:
It's not ignored because it's observable from comptime code, for example in `lib/std/start.zig`.

Reply:
Huh, okay.